okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/OkHttpClients.java

```diff
@@ -60,6 +64,12 @@
     private static final ScheduledExecutorService scheduledExecutorService = Tracers.wrap(
             new ScheduledThreadPoolExecutor(5, Util.threadFactory("http-remoting/OkHttp Scheduler", false)));
 
+    /**
+     * The per service and host metrics recorded for each HTTP call.
+     */
+    private static final HostMetricsRegistry hostMetrics =
```

Comment:
would be awesome if could inject this. possible?

Reply:
if we wanna start passing around instances of `OkHttpClients` and deprecate the static `create` method. I'm not convinced being able to pass one of these in is particularly useful?